site/source/support/faq/faq-basic-use.rst

```diff
@@ -71,11 +67,11 @@ No, you don't need to delete the old backups. The technology we use updates the
 
 Can I clone my Embassy SSD for backup purposes?
 -----------------------------------------------
-.. warning:: **DO NOT do this if you are running LND or c-lightning**. If you clone the SSD, then go back to running LND or c-lightning, and you *ever* try to restore the SSD, there is a good chance you will lose *all your channel funds*. Also, if you try to use the SSD for a 2nd Embassy, that will also result in loss of funds. This has nothing to do with Start9 or the Embassy; it is inherent to the architecture of Lightning.
+.. warning:: **DO NOT do this if you are running LND or CLN**. If you clone the SSD, then go back to running LND or CLN, and you *ever* try to restore the SSD, there is a good chance you will lose *all your channel funds*. Also, if you try to use the SSD for a 2nd Embassy, that will also result in loss of funds. This has nothing to do with Start9 or the Embassy; it is inherent to the architecture of Lightning.
 
 It is neither necessary, nor recommended to do this.  The backup solution available in EmbassyOS (v0.3.0+) allows all data to be written to another drive remotely, and this can mean another SSD, attached to a computer on your network.  You may also do a backup to a local drive if it has external power.  Please follow the directions in the backup UI for more details.
 
-Why would I even buy this when I can just build it for free??
+Why would I even buy this when I can just build it for free?
 -------------------------------------------------------------
 (1) White glove support. Because each Embassy comes with a unique product key engraved on it, and we have a record of all product keys, we can ask the user to verify their product key in order to receive a higher tier of support, such as phone calls.
```

Comment:
There will no longer be engraved product keys, furthermore, we actually need to search docs for all references to product keys and strike them, with the exception of one FAQ along the lines of - "I bought an Embassy before Q3 2022 and it came with a 'product key,' what is that?"